src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total / perPage);
+  const pages = [];
```

Comment:
You can create pages with helper `getNumbers`

```suggestion
  const pages = getNumbers(1, total/perPage);
```